content/guides/depin/getting-started.md

```diff
@@ -0,0 +1,209 @@
+---
+featured: false
+featuredPriority: 1
+date: 2024-12-02T00:00:00Z
+difficulty: beginner
+seoTitle: "DePIN Quickstart Guide"
+title: "DePIN Quickstart Guide"
+description:
+  "While each DePIN network has a unique product focus, most DePIN networks
+  utilize Solana for a common set of use-cases. This guide is meant to help
+  builders get oriented to common onchain DePIN use-cases."
+keywords:
+  - DePIN
+  - token
+  - governance
+tags:
+  - depin
+  - governance
+  - zk compression
+---
+
+DePIN (Decentralized Physical Infrastructure Network) refers to blockchain-based
+networks that incentivize the deployment, operation, and sharing of real-world
+physical infrastructure through decentralized mechanisms. These networks
+leverage tokens to reward participants for contributing to and maintaining the
+infrastructure, creating a decentralized alternative to traditional centralized
+systems. Examples include wireless, content delivery, and mapping networks.
+
+While each DePIN network has a unique product focus, most DePIN networks utilize
+Solana for a common set of use cases. This guide is meant to help builders get
+oriented to these common on-chain DePIN use cases. Topics covered include:
+
+- [Minting and Managing a Token](#minting-and-managing-a-token)
+- [Rewards Distribution](#rewards-distribution)
+- [Proof of Contribution](#proof-of-contribution)
+- [Governance](#governance)
+
+## Minting and Managing a Token
+
+### Token Minting
+
+When minting a token on Solana, there are two token programs to choose from: the
+[Token program](https://spl.solana.com/token) or the
+[Token22 program](https://spl.solana.com/token-2022). There are tradeoffs to
+consider (discussed
+[here](https://solana.stackexchange.com/questions/9205/what-is-the-advantage-of-using-the-token22-token-extensions-program-over-the-old)).
+The recommended selection between the two options ultimately reduces to whether
+the features in the token extensions program would be of use to the application.
+
+### Token Listing
+
+In order to have your token appear in explorers, decentralized exchanges, and
+other ecosystem token lists,
+[getting the token verified through Jupiter Verification](https://station.jup.ag/guides/general/get-your-token-on-jupiter)
+is recommended.
+
+### Modeling Token Distribution
+
+For DePIN teams looking to optimize the way they allocate their token across
+various needs and stakeholders (rewards for participating / supplying network
+resources, private or public token sales etc.), tools like
+[Forgd](https://www.forgd.com/) can be used to model out and refine token
+distribution strategies.
+
+### Token Management
+
+Most teams utilize [Squads](https://squads.so/) for their onchain treasury
+management, also leveraging their multisig feature.
```

Comment:
"also leveraging their multisig features" -> "and leverage features such as multi-sig and time-based lockups"